scenario_specifications.tex

```diff
@@ -1,54 +1,49 @@
 \section{Scenario Specifications}
 
-The simulation is run from year 1970 to 2160. Year 2030 is chosen for the 
-SFR development consideration.
-
+\subsection{Past - Predictable Future}
 \begin{itemize}
-	\item \textbf{Simulation Time:}  1970-2160 
-	\item \textbf{\gls{SFR} Available Year:}  2030  
+	\item \textbf{Simulation Time:}  1970-2050 
```

Comment:
I originally tried it as a table, but since the length of the cells are different, it hurts the readability..

Is there a way to create a table without sacrificing the readability?